src/main/java/frc/trigon/robot/subsystems/arm/kablamaArm/KablamaArmIO.java

```diff
@@ -0,0 +1,86 @@
+package frc.trigon.robot.subsystems.arm.kablamaArm;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import frc.trigon.robot.subsystems.arm.ArmIO;
+import frc.trigon.robot.subsystems.arm.ArmInputsAutoLogged;
+import frc.trigon.robot.utilities.Conversions;
+
+public class KablamaArmIO extends ArmIO {
+    private final CANSparkMax
+            masterAngleMotor = KablamaArmConstants.MASTER_ANGLE_MOTOR,
+            followerAngleMotor = KablamaArmConstants.FOLLOWER_ANGLE_MOTOR,
+            masterElevatorMotor = KablamaArmConstants.MASTER_ELEVATOR_MOTOR,
+            followerElevatorMotor = KablamaArmConstants.FOLLOWER_ELEVATOR_MOTOR;
+
+    @Override
+    protected void updateInputs(ArmInputsAutoLogged inputs) {
+        inputs.angleMotorVoltage = masterAngleMotor.getBusVoltage();
+        inputs.angleMotorCurrent = masterAngleMotor.getOutputCurrent();
+        inputs.elevatorMotorVoltage = masterElevatorMotor.getBusVoltage();
+        inputs.elevatorMotorCurrent = masterElevatorMotor.getOutputCurrent();
+        inputs.elevatorPositionRevolution = getElevatorPositionRevolutions();
+        inputs.angleVelocityDegreesPerSecond = getAngleVelocityDegreesPerSecond();
+        inputs.elevatorVelocityRevolutionsPerSecond = getElevatorVelocityRevolutionsPerSecond();
+        inputs.anglePositionDegrees = getAngleMotorPositionDegrees().getDegrees();
+    }
```

Comment:
Have a whitespace here between each mechanism motors, update these in the same order you declare them in the class